<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarketTradeView</class>
 <widget class="QWidget" name="MarketTradeView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1260</width>
    <height>764</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="2" column="0">
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelMarketID">
        <property name="text">
         <string>Market ID:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelMarketIDValue">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCopyMarketID">
        <property name="text">
         <string>Copy</string>
        </property>
        <property name="icon">
         <iconset resource="../hivemind.qrc">
          <normaloff>:/icons/editcopy</normaloff>:/icons/editcopy</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabStandard">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <attribute name="title">
       <string>Standard</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QFrame" name="frameMarketGraph">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Market Graph:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="1" column="1">
              <widget class="QLabel" name="labelMarketPriceValue">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QRadioButton" name="radioButtonMinutes">
               <property name="text">
                <string>5 Minutes</string>
               </property>
               <property name="checked">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="labelMarketPrice">
               <property name="text">
                <string>Market Price:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QRadioButton" name="radioButtonDay">
               <property name="text">
                <string>1 Day</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QRadioButton" name="radioButtonMonth">
               <property name="text">
                <string>1 Month</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="labelOwned">
               <property name="text">
                <string>Shares owned:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="labelOwnedValue">
               <property name="text">
                <string>0</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Your trades:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidgetYourTrades"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QFrame" name="frame_10">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QRadioButton" name="radioButtonLong">
               <property name="text">
                <string>Long (Buy)</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButtonShort">
               <property name="text">
                <string>Short (Sell)</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QTabWidget" name="tabWidgetTradeTypes">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="currentIndex">
             <number>2</number>
            </property>
            <widget class="QWidget" name="tabSetMarketPrice">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <attribute name="title">
              <string>Set Market Price</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QFrame" name="frame_7">
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QLabel" name="labelCurrentMarketPrice">
                   <property name="text">
                    <string>Current Price:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="labelCurrentPriceValue">
                   <property name="text">
                    <string>1234.012</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="Line" name="line_4">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="labelSetPrice">
                   <property name="text">
                    <string>Set Price to:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QDoubleSpinBox" name="doubleSpinBoxSetPrice"/>
                 </item>
                 <item>
                  <spacer name="verticalSpacer_2">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tabQuickBuy">
             <attribute name="title">
              <string>Quick Buy</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QFrame" name="frame_8">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <widget class="QGroupBox" name="groupBox">
                   <property name="title">
                    <string>Preset</string>
                   </property>
                   <layout class="QGridLayout" name="gridLayout">
                    <item row="0" column="1">
                     <widget class="QPushButton" name="pushButtonBuy5">
                      <property name="text">
                       <string>5</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="0">
                     <widget class="QPushButton" name="pushButtonBuy20">
                      <property name="text">
                       <string>20</string>
                      </property>
                     </widget>
                    </item>
                    <item row="2" column="1">
                     <widget class="QPushButton" name="pushButtonBuy50">
                      <property name="text">
                       <string>50</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="1">
                     <widget class="QPushButton" name="pushButtonBuy15">
                      <property name="text">
                       <string>15</string>
                      </property>
                     </widget>
                    </item>
                    <item row="0" column="0">
                     <widget class="QPushButton" name="pushButtonBuy0">
                      <property name="text">
                       <string>0</string>
                      </property>
                     </widget>
                    </item>
                    <item row="1" column="0">
                     <widget class="QPushButton" name="pushButtonBuy10">
                      <property name="text">
                       <string>10</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="0">
                     <widget class="QPushButton" name="pushButtonBuy100">
                      <property name="text">
                       <string>100</string>
                      </property>
                     </widget>
                    </item>
                    <item row="3" column="1">
                     <widget class="QPushButton" name="pushButtonBuy250">
                      <property name="text">
                       <string>250</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </widget>
                 </item>
                 <item>
                  <spacer name="verticalSpacer_3">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tabMakeOrder">
             <attribute name="title">
              <string>Make Order</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QFrame" name="frame_9">
                <property name="frameShape">
                 <enum>QFrame::NoFrame</enum>
                </property>
                <property name="frameShadow">
                 <enum>QFrame::Raised</enum>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <item>
                  <widget class="QLabel" name="labelShares">
                   <property name="text">
                    <string># Shares:</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QSpinBox" name="spinBoxShares"/>
                 </item>
                 <item>
                  <spacer name="verticalSpacer_4">
                   <property name="orientation">
                    <enum>Qt::Vertical</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>20</width>
                     <height>40</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tabHelp">
             <attribute name="icon">
              <iconset resource="../hivemind.qrc">
               <normaloff>:/icons/transaction_0</normaloff>:/icons/transaction_0</iconset>
             </attribute>
             <attribute name="title">
              <string>Help</string>
             </attribute>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Trade Info:</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_4">
             <item row="4" column="0">
              <widget class="QFrame" name="frame_5">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="labelSharesToBuy">
                  <property name="text">
                   <string>Shares to buy:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelSharesToBuyValue">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelFunds">
                  <property name="text">
                   <string>Balance:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelFundsValue">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelCost">
                  <property name="text">
                   <string>Trade Cost:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="labelCostValue">
                  <property name="text">
                   <string>0</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLineEdit" name="lineEditAddress"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="labelDecisionState">
               <property name="text">
                <string>Decision State:</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="labelAddress">
               <property name="text">
                <string>Address:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QSpinBox" name="spinBoxState"/>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonFinalize">
            <property name="text">
             <string>Finalize</string>
            </property>
            <property name="icon">
             <iconset resource="../hivemind.qrc">
              <normaloff>:/icons/transaction_confirmed</normaloff>:/icons/transaction_confirmed</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTwoDimensional">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>Two Dimensional</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tabHighDimensional">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>High Dimensional</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../hivemind.qrc"/>
 </resources>
 <connections/>
</ui>
